vimrc.bundles

```diff
@@ -43,6 +45,8 @@ Plug 'tpope/vim-repeat'
 Plug 'tpope/vim-surround'
 Plug 'vim-ruby/vim-ruby'
 Plug 'vim-scripts/tComment'
+Plug 'nomadcoder/smyck'
```

Comment:
What do you think about installing solarized as well and then everyone can easily customize or alternate:

`Plug 'altercation/vim-colors-solarized'`